examples/plot_background_model.py

```diff
@@ -1,99 +1,18 @@
-        hdu_list = fits.open(filename)
-        hdu = hdu_list['BACKGROUND']
-        data = hdu.data
-
-        det_bins = _make_bin_edges_array(data['DETX_LO'], data['DETX_HI'])
-        det_bins = Angle(det_bins, 'deg')
-        energy_bins = _make_bin_edges_array(data['ENERG_LO'], data['ENERG_HI'])
-        energy_bins = Quantity(energy_bins, 'TeV')
-        background = data['Bgd'][0]
-
-        return CubeBackgroundModel(det_bins=det_bins,
-                                   energy_bins=energy_bins,
-                                   background=background)
-
-    @property
-    def image_extent(self):
-        """Image extent `(x_lo, x_hi, y_lo, y_hi)` in deg."""
-        b = self.det_bins.degree
-        return [b[0], b[-1], b[0], b[-1]]
-
-
-    def plot_images(self, filename=None):
-        import matplotlib.pyplot as plt
-
-        nimages = len(self.energy_bins) - 1
-        ncols = int(np.sqrt(nimages)) + 1
-        nrows = (nimages // ncols) + 1
-        # print(nimages, ncols, nrows)
-
-        fig, axes = plt.subplots(nrows=nrows, ncols=ncols)
-
-        extent = self.image_extent
-
-        for ii in range(nimages):
-            data = self.background[ii]
-            energy = self.energy_bins[ii]
-            ax = axes.flat[ii]
-            image = ax.imshow(data, extent=extent, interpolation='nearest',
-                              cmap='afmhot')
-            ax.set_title('Energy = {:.1f}'.format(energy))
-            # fig.colorbar(image)
-        # import IPython; IPython.embed()
-
-        if filename:
-            print('Wrinting {}'.format(filename))
-            fig.savefig(filename)
-
-    def plot_spectra(self, filename):
-        raise NotImplementedError
-
-    def write_cube(self, filename):
-        hdu = fits.ImageHDU(data=self.background)
-        print('Writing {}'.format(filename))
-        hdu.writeto(filename, clobber=True)
-
+from gammapy.background.models import CubeBackgroundModel
+from gammapy import datasets
 
 def plot_example():
-    DIR = '/Users/deil/work/_Data/hess/HESSFITS/pa/Model_Deconvoluted_Prod26/Mpp_Std/background/'
-    filename = DIR + 'hist_alt3_az0.fits.gz'
-    bg_model = CubeBackgroundModel.read(filename)
-    bg_model.plot_images('cube_background_model.png')
-    bg_model.write_cube('cube_background_model.fits')
+    filename = '../test_datasets/background/bg_cube_model_test.fits'
+    filename = datasets.get_path(filename, location='remote')
+    bg_model = CubeBackgroundModel.read_bin_table(filename)
+    print("Plotting all images and spectra (can take a couple of mins).")
+    bg_model.plot_images()
+    bg_model.plot_spectra()
+    outname = 'cube_background_model'
+    bg_model.write_bin_table('{}_bin_table.fits'.format(outname), clobber=True)
+    bg_model.write_image('{}_image.fits'.format(outname), clobber=True)
 
 if __name__ == '__main__':
-    plot_example()
\ No newline at end of file
+    plot_example()
```

Comment:
Now that this example script is very short, I'd suggest to just make it top-level statements,
i.e. remove the function and `if __name__ == '__main__'`.

It would also be great to have an example in the docs.
Maybe include it from there? (it's possible with a relative path `../examples/...`